File: 2012 Book Source Code/Chapter_7_Source_Code/Chapter 7/Employee Class Tester/Tester.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Pro_SSRS
{
	public partial class Tester : Form
	{
		public Tester()
		{
			InitializeComponent();
		}

		private void xmlTest_Click(object sender, EventArgs e)
		{
			string employeeID;
			DateTime visitDate;
			try
			{
				employeeID = EmployeeID.Text;
				visitDate = Convert.ToDateTime(VisitDate.Text);
				EmployeePayAmt.Text = Employee.CostPerVisitXML(employeeID, visitDate).ToString("C");
			}
			catch (Exception)
			{
				throw;
			}
		}

		private void wsTest_Click(object sender, EventArgs e)
		{
			string employeeID;
			DateTime visitDate;
			try
			{
				employeeID = EmployeeID.Text;
				visitDate = Convert.ToDateTime(VisitDate.Text);
				EmployeePayAmt.Text = Employee.CostPerVisitWS(employeeID, visitDate).ToString("C");
			}
			catch (Exception)
			{
				throw;
			}
		}
	}
}

File: 2012 Book Source Code/Chapter_7_Source_Code/Chapter 7/Employee Class Tester VB/Tester.vb
Public Class Tester

    Private Sub xmlTest_Click(sender As System.Object, e As System.EventArgs) Handles xmlTest.Click
        Dim empID As String
        Dim visDate As DateTime

        Try
            empID = EmployeeID.Text
            visDate = Convert.ToDateTime(VisitDate.Text)
            EmployeePayAmt.Text = Employee_VB.Employee.CostPerVisitXML(empID, visDate).ToString("C")


        Catch ex As Exception
            Throw
        End Try
    End Sub

    Private Sub wsTest_Click(sender As System.Object, e As System.EventArgs) Handles wsTest.Click
        Dim empID As String
        Dim visDate As DateTime

        Try
            empID = EmployeeID.Text
            visDate = Convert.ToDateTime(VisitDate.Text)
            EmployeePayAmt.Text = Employee_VB.Employee.CostPerVisitWS(empID, visDate).ToString("C")
        Catch ex As Exception
            Throw
        End Try
    End Sub
End Class
